packages/@react-aria/select/src/HiddenSelect.tsx

```diff
@@ -93,8 +98,8 @@ export function useHiddenSelect<T>(props: AriaHiddenSelectOptions, state: Select
       tabIndex: -1,
       autoComplete,
       disabled: isDisabled,
+      required: validationBehavior === 'native' && isRequired,
       name,
-      size: state.collection.size,
```

Comment:
were we just not using this?

Reply:
looks like it was added for testing purposes, but `<select>` with a `size` does not participate in form validation so had to remove it.